src/main/java/com/google/sps/daos/DatabaseGameInstanceDao.java

```diff
@@ -19,25 +19,25 @@
 import com.google.firestore.v1.StructuredQuery.CollectionSelector;
 
 import com.google.sps.data.Member;
-import com.google.sps.data.Room;
+import com.google.sps.data.GameInstance;
 
-public class DatabaseRoomDao implements RoomDao {
+public class DatabaseGameInstanceDao implements GameInstanceDao {
 
   private Firestore firestoreDb;
-  private static Logger log = Logger.getLogger(DatabaseRoomDao.class.getName());
+  private static Logger log = Logger.getLogger(DatabaseGameInstanceDao.class.getName());
 
-  public DatabaseRoomDao(Firestore firestoreDb) {
+  public DatabaseGameInstanceDao(Firestore firestoreDb) {
     this.firestoreDb = firestoreDb;
   }
   
   @Override
-  public Room createNewRoom(String gameId) {
+  public GameInstance createNewRoom(String gameId) {
 
     DocumentReference newRoomRef = firestoreDb.collection("liveRooms").document();
```

Comment:
And also change the name in the db so the collection will be gameInstance

Reply:
Yes you re right